src/viewer/proxy/CMakeLists.txt

```diff
@@ -24,11 +26,13 @@ LINK_DIRECTORIES(${LIBRARY_OUTPUT_DIRECTORY})
 ADD_LIBRARY(viewerproxy ${VIEWERPROXY_SOURCES})
 TARGET_LINK_LIBRARIES(viewerproxy viewerrpc visitcommon avtdbatts)
 
-ADD_EXECUTABLE(proxyexample proxyexample.C)
-set_target_properties(proxyexample PROPERTIES
-                      FOLDER examples
-                      EXCLUDE_FROM_ALL 1
-                      EXCLUDE_FROM_DEFAULT_BUILD 1)
-TARGET_LINK_LIBRARIES(proxyexample viewerrpc viewerproxy visitcommon avtdbatts)
+if(VISIT_ENABLE_EXAMPLES)
```

Comment:
Ok, so here it appears that this is not just an `AVT` example. Do we know what these examples are? Are they documented anywhere? Not suggesting you take that on. But, maybe add an issue ticket if not.